ftw/file/content/dxfile.py

```diff
@@ -41,6 +41,11 @@ def __init__(self, data='', contentType='', filename=None):
             if mimetypeitem:
                 contentType = mimetypeitem.normalized()
 
+        # Products.MimetypesRegistry 2.0.8 returned unicode afterwards it was utf-8
+        # The field contentType, however requires utf-8, since its a zope.schema.BytesLine
+        if isinstance(contentType, unicode):
+            contentType = contentType.encode('utf8')
```

Comment:
I think this change could be indented to be within `if mimetypeitem:`, but perhaps better safe than sorry?

Reply:
I would let the code there, so if anybody else calls this, wich the contentType as unicode it gets fixed right away

Reply:
ok